code/style.css

```diff
@@ -1,4 +1,182 @@
-html {
-  background: #0025ff;
+body {
+  background-color: rgb(38, 77, 148);
   color: #fff;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin: 0;
+}
+
+/***HEADER STYLING ***/
+
+header {
+  background-color: rgb(128, 38, 83);
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+}
+
+nav {
+display: none;
+ 
+}
+
+nav a {
+  color: #fff;
+  text-decoration: none;
+  margin-left: 16px;
+  font-size: 20px;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+  
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #fff;
+  border-radius: 5px;
+}
+
+header img {
+width: 50px;
+height: 50px;
+}
+
+header p {
+  font-size: 30px;
+}
+
+/***BIG NEWS CONTENT***/
+.big-news img {
+  width: 100%;
+}
+
+.big-news {
+  display: flex;
+  flex-direction: column;
+  align-items: center;  
+}
+
+.big-news div {
+  text-align: center;
+  }
+ 
+
+/***NEWS CONTENT***/
+.news-container {
+  display: flex;
+  flex-direction: column;  
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+}
+
+.news-container img {
+   width: 330px;
+   height: 400px;
+   object-fit: cover;
+}
```

Comment:
Be careful with using fixed widths like: `width: 330px;`. This will make the site look a little bit wonky if a user is on an older, smaller device like an iPhone 5. It's no worries right now but good to have in mind for the future ✌️